### PR TITLE
Add translation key for 404 image title

### PR DESCRIPTION
**Pull request by Mysterious-Dev**
This PR add key for 404 image title to allow translated.

### CODE REVIEW

**Comment by Mysterious-Dev** [comment]
Honestly, I hesitated to include a translation key for the image alt...

**Review by Southpaw1496 — APPROVED** [review]
Looks fine to me, but I'll wait for an OK from someone more familiar with localisation before merging.

**Review by pluiedev — APPROVED** [review]

**Pull request merged**
